gammapy/datasets/fermi.py

```diff
@@ -216,6 +218,166 @@ def fetch_fermi_diffuse_background_model(filename='gll_iem_v02.fit'):
 
     return filename
 
+class Fermi3FGLObject(object):
+    """TODO: Doccomments
+    """
+
+    from astropy.units import Quantity
+    from gammapy.spectrum import energy_bin_centers_log_spacing
```

Comment:
When you rebase against master, you'll get this error:
```
ImportError: cannot import name 'energy_bin_centers_log_spacing'
```
This is something that Johannes recently improved. See [here](https://gammapy.readthedocs.org/en/latest/spectrum/index.html#energy-handling-in-gammapy) for a high-level description ... I think you want to call [gammapy.spectrum.Energy.equal_log_spacing](https://gammapy.readthedocs.org/en/latest/api/gammapy.spectrum.Energy.html#gammapy.spectrum.Energy.equal_log_spacing) here. Please try and comment if there are any issues or it's not clear how to do it.